tests/test_recruiters.py

```diff
@@ -405,18 +405,27 @@ def test_open_recruitment_with_blacklist(self, recruiter):
             annotation='some experiment uid',
         )
 
-    def test_open_recruitment_is_noop_if_experiment_in_progress(self, a, recruiter):
-        a.participant()
-        recruiter.open_recruitment()
+    def test_open_recruitment_raises_error_if_recruitment_in_progress(self, a, recruiter):
+        a.participant(recruiter_id='mturk')
+        with pytest.raises(RuntimeError):
+            recruiter.open_recruitment()
 
         recruiter.mturkservice.check_credentials.assert_not_called()
 
+    def test_open_recruitment_ignores_participants_from_other_recruiters(self, a, recruiter):
+        a.participant(recruiter_id='bot')
+        result = recruiter.open_recruitment(n=1)
+        assert len(result['items']) == 1
+        recruiter.mturkservice.check_credentials.assert_called_once()
+
     def test_supresses_assignment_submitted(self, recruiter):
         assert recruiter.submitted_event() is None
 
     def test_current_hit_id_with_active_experiment(self, a, recruiter):
-        a.participant(hit_id=u'the hit!')
+        a.participant(hit_id=u'not the hit!', recruiter_id='hotair')
```

Comment:
I got confused thinking the hit IDs were the significant in a way that they aren't.
Could we use the same HIT Id in both cases, since it's the recruiter_id distinction that matters?:
`a.participant(hit_id=u'some hit ID', recruiter_id='hotair')` and `a.participant(hit_id=u'some hit ID', recruiter_id='mturk')`